server/events/command_context.go

```diff
@@ -49,4 +49,6 @@ type CommandContext struct {
 	PullStatus *models.PullStatus
 
 	Trigger CommandTrigger
+
+	TemplateOverrides map[string]string
```

Comment:
This doesn't need to be in the command context as it's not changing between requests, this should probably just be injected into the MarkdownRenderer as part of its construction on server startup.

Reply:
Agreed. Removed this from the command context.